### ISSUE TITLE
Review ratings not keyboard accessible

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#12929]
### Shopware Version

6.7.3.0

### Affected area / extension

Platform(Default)

### Actual behaviour

It is currently not possible to change the review stars with a keyboard which is a failure of [WCAG 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html)

https://github.com/user-attachments/assets/270592c0-8151-44dc-a28f-5d10d0c7cd22

### Expected behaviour

I should be able to change the star rating with a mouse, keyboard, or screenreader.

### How to reproduce

Try to change the Excellent 5 star rating using only a keyboard.
